src/utils/backend/person/teacher.ts

```diff
@@ -264,3 +268,62 @@ export async function getClassAdvisorAt(
 
   return { data: data!, error: null };
 }
+
+export async function getOnBoardTeacherData(
```

Comment:
“Onboard” is one word, so a better name would be `getTeacherOnboardStatus`, or something like that.